rero_ils/config.py

```diff
@@ -3639,3 +3642,183 @@ def _(x):
 RERO_ILS_PASSWORD_SPECIAL_CHAR = False
 RERO_ILS_PASSWORD_GENERATOR = 'rero_ils.modules.utils:password_generator'
 RERO_ILS_PASSWORD_VALIDATOR = 'rero_ils.modules.utils:password_validator'
+
+# ADVANCED SEARCH CONFIG
+# ======================
+def search_type(field):
+    """Search type options.
+
+    :param: field: the field key.
+    :return: a list of options.
+    """
+    output = [];
+    if field not in [
+        'country', 'canton', 'rdaContentType', 'rdaMediaType',
+        'rdaCarrierType'
+    ]:
+        output.append({'label': _('contains'), 'value': 'contains'})
+    output.append({'label': _('exact'), 'value': 'phrase'})
```

Comment:
Fields "isbn", "issn", and "identifier" should only have option "contains". A phrase search makes no sense for these fields.

Reply:
Fixed